multiapps-controller-web/src/test/java/org/cloudfoundry/multiapps/controller/web/security/AuthorizationCheckerTest.java

```diff
@@ -74,26 +73,26 @@ void setUp() throws Exception {
     static Stream<Arguments> checkPermissionUsingGuidsTest() {
         return Stream.of(
                          // (0) User has a space developer role and executes a non read-only request
-                         Arguments.of(List.of(UserRole.SpaceRole.SPACE_DEVELOPER), false, true),
+                         Arguments.of(List.of(UserRole.SPACE_DEVELOPER), false, true),
                          // (1) User does not have any roles and executes a non read-only request
                          Arguments.of(Collections.emptyList(), false, false),
                          // (2) User does not have any roles and executes a read-only request
                          Arguments.of(Collections.emptyList(), true, false),
                          // (3) User has a space auditor role and executes a non read-only request
-                         Arguments.of(List.of(UserRole.SpaceRole.SPACE_AUDITOR), false, false),
+                         Arguments.of(List.of(UserRole.SPACE_AUDITOR), false, false),
                          // (4) User has a space manager role and executes a non read-only request
-                         Arguments.of(List.of(UserRole.SpaceRole.SPACE_MANAGER), false, false),
+                         Arguments.of(List.of(UserRole.SPACE_MANAGER), false, false),
                          // (5) User has a space auditor role and executes a read-only request
-                         Arguments.of(List.of(UserRole.SpaceRole.SPACE_AUDITOR), true, true),
+                         Arguments.of(List.of(UserRole.SPACE_AUDITOR), true, true),
                          // (6) User has a space manager role and executes a read-only request
-                         Arguments.of(List.of(UserRole.SpaceRole.SPACE_MANAGER), true, true),
+                         Arguments.of(List.of(UserRole.SPACE_MANAGER), true, true),
                          // (7) User has a space developer role and executes a read-only request
-                         Arguments.of(List.of(UserRole.SpaceRole.SPACE_DEVELOPER), true, true));
+                         Arguments.of(List.of(UserRole.SPACE_DEVELOPER), true, true));
```

Comment:
Can add new test cases when list contains more than one element.
e.g. [SpaceDeveloper, SpaceAuditor] and [SpaceAuditor, SpaceManager]